stable/sentry/templates/metrics-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-metrics
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+        role: metrics
+    spec:
+      {{- if .Values.metrics.affinity }}
+      affinity:
+{{ toYaml .Values.metrics.affinity | indent 8 }}
+      {{- end }}
+      {{- if .Values.metrics.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.metrics.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.metrics.tolerations }}
+      tolerations:
+{{ toYaml .Values.metrics.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.metrics.schedulerName }}
+      schedulerName: "{{ .Values.metrics.schedulerName }}"
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}-metrics
+        args: 
+          - "--statsd.listen-udp=:9125"
+          - "--web.listen-address=:9102"
+        image: "{{ .Values.metrics.image.repository }}:{{ .Values.metrics.image.tag }}"
+        imagePullPolicy: {{ .Values.metrics.image.pullPolicy }}
+        ports:
+        - name: statsd
+          containerPort: 9125
+        - name: metrics
+          containerPort: 9102
+        livenessProbe:
```

Comment:
I believe it should be configurable. 
Something like this https://github.com/okgolove/helm-charts/blob/master/stable/asprom/templates/deployment.yaml#L41

Reply:
Ok done